land_use/serializers.py

```diff
@@ -0,0 +1,88 @@
+from decimal import Clamped
+from pyexpat import model
+from django.db import models
+from rest_framework import serializers
+from rest_framework_gis import serializers as gis_serializers
+
+from land_use import models
+
+
+class LandUseSerializer(gis_serializers.GeoFeatureModelSerializer):
+    """Serializer for geographic `models.LandUseClasses` spatial data."""
+    class Meta:
+        """Meta calss for geographic data `LandUseSerializer` serializer."""
+        model = models.LandUseClasses
+        id_field = False
+        geo_field = 'geom'
+        fields = (
+            'id',
+            'geom',
```

Comment:
Please, remove it from fields.
This data is already on FeatureCollection, you're duplicating the data.

Reply:
Adjusted.